api/wip-files/examples/2024-06-10-preview/HypershiftClusters_CreateOrUpdate.json

```diff
@@ -0,0 +1,76 @@
+{
+  "title": "HypershiftClusters_CreateOrUpdate",
```

Comment:
please rename the example files to match the resourceType name?

Reply:
I have removed the examples, will create new ones once the API is acked.